index.css

```diff
@@ -385,6 +350,153 @@ li {
   margin-top: 1em;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  border-radius: 20px;
+  z-index: 10;
+  top: 0;
+  width: 90%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+  left: 0;
+  margin: 0;
+}
```

Comment:
- [ ] Great job for implementing popup window in your project. You've really done a very wonderful job. However, there're few things you need to put in place before you could move on to the next milestone. You must also stick to the figma design as much as possible. Kindly check the two screenshots below and work your design alongside the figma design.

#### Your Modal Desktop
![your-card-1](https://user-images.githubusercontent.com/26597432/223992372-01f1f664-30e6-47cc-967b-c03fd7daeb22.png)

###  Figma Modal Desktop
![temp-card](https://user-images.githubusercontent.com/26597432/223992429-e3434d03-9324-42f0-89ea-0aa76d654f81.png)